website/static/js/institutionNodes.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var ko = require('knockout');
+var Raven = require('raven-js');
+
+var $osf = require('js/osfHelpers');
+
+
+var ViewModel = function(context) {
+    var self = this;
+    self.ctx = context;
+    self.allNodes = ko.observable();
```

Comment:
OK, but it's still a good idea to make this an `observableArray`. (1) It communicates that the value of this observable is an array and (2) prevents future devs from becoming sad if they ever do `self.allNodes.push(node)`.

All that said, good call on assigning the whole list at once; that is preferred over `push` anyway, so :+1: